docs/source/api/pipelines/stable_diffusion.mdx

```diff
@@ -31,6 +31,30 @@ For more details about how Stable Diffusion works and how it differs from the ba
 
 ## Tips
 
+### How to load and use different schedulers.
+
+The stable diffusion pipeline uses [`PNDMScheduler`] scheduler by default. But `diffusers` provides many other schedulers that can be used with the stable diffusion pipeline such as [`DDIMScheduler`], [`LMSDiscreteScheduler`], [`EulerDiscreteScheduler`], [`EulerAncestralDiscreteScheduler`] etc.
+To use a different scheduler, you can pass the `scheduler` argument to `from_pretrained` method of the pipeline. For example, to use the [`LMSDiscreteScheduler`], you can do the following:
+
+```python
+from diffusers import StableDiffusionPipeline, LMSDiscreteScheduler
+
+lms_scheduler = LMSDiscreteScheduler.from_config("CompVis/stable-diffusion-v1-4", subfolder="scheduler")
+pipeline = StableDiffusionPipeline.from_pretrained("CompVis/stable-diffusion-v1-4", scheduler=lms_scheduler)
+```
+
+It's also possible to hot swap the `scheduler` by setting the `scheduler` attribute of the already loaded pipeline. For example, to use the [`EulerDiscreteScheduler`] with the pipeline loaded above, you can do the following:
```

Comment:
```suggestion
```

Actually let's not advertise this as it'll currently fail and in general I don't think it's a good idea (because doing this will mean that saved config of the pipeline is wrong). 

All this code: https://github.com/huggingface/diffusers/blob/a793b1fe7e05adaa74f35193a6c3119265c6d889/src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py#L83 will then not be run on the scheduler which a) can mean that the config will be wrong leading to bad results and b) the saved config of the pipeline is wrong, *e.g.* `pipeline.config` would show an incorrect scheduler.

Reply:
Agree! Removed it from the docs.